release-notes/6.0/supported-os.md

```diff
@@ -31,11 +31,12 @@ OS                                    | Version                 | Architectures
 OS                                    | Version               | Architectures     | Lifecycle
 --------------------------------------|-----------------------|-------------------|----------
 [Alpine Linux][Alpine]                | 3.13+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
-[CentOS][CentOS]                      | 7+                    | x64               | [CentOS][CentOS-lifecycle]
+[CentOS Linux][CentOS]                | 7                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS Stream Linux][CentOS]         | 8                     | x64               | [CentOS][CentOS-lifecycle]
```

Comment:
Should it be "8+"?  [CentOS Stream 9](https://centos.org/stream9/) exists too.

Reply:
Goot catch. @leecow?

Reply:
Should be `CentOS Stream` instead of `CentOS Stream Linux` (we also don't write `Ubuntu` as `Ubuntu Linux` in the list) and 8+ for the version as also caught by @omajid for .NET 6 and .NET 7.

https://learn.microsoft.com/en-us/dotnet/core/install/linux-rhel#supported-distributions has it correctly.

Reply:
For .NET 7, RHEL 7 is also still shown as supported. Since CentOS Stream is showing 8+, RHEL should also be fixed :)

Reply:
Thanks, addressing comments in https://github.com/dotnet/core/pull/8386